elastalert/test_rule.py

```diff
@@ -349,6 +349,17 @@ def run_rule_test(self):
             action='store',
             dest='save',
             help='A file to which documents from the last day or --days will be saved')
+        parser.add_argument(
+            '--use-downloaded',
+            action='store_true',
+            dest='use_downloaded',
+            help='Use the downloaded '
+        )
+        parser.add_argument(
```

Comment:
Can you add a default to this? I think Elasticsearch defaults to 10 , which I think is much too small for this use case

Reply:
The default was 10000, so I'll make that the default.